integreat_cms/cms/models/abstract_tree_node.py

```diff
@@ -230,41 +228,29 @@ def move(self, target: AbstractTreeNode, pos: str | None = None) -> None:
         :param pos: The new position of the page relative to the target
                     (choices: :mod:`~integreat_cms.cms.constants.position`)
         :raises ~treebeard.exceptions.InvalidPosition: If the node is moved to another region
-        :raises ~db_mutex.exceptions.DBMutexError: If the DB mutex could not be retrieved
-        :raises ~db_mutex.exceptions.DBMutexTimeoutError: If waiting for the DB mutex timed out
         """
         logger.debug("Moving %r to position %r of %r", self, pos, target)
-        try:
```

Comment:
We do not need the mutex here anymore, right? We directly implement them in the views.